events.html

```diff
@@ -145,7 +145,7 @@
         />
       </a>
       <section class="event-details">
-        <h2 class="event-upcoming">Upcoming Events</h2>
+        <h2 class="event-upcoming">Recap of our past events</h2>
```

Comment:
There are mulitple issues mentioned in the ticket, but there's very small change here.
Not getting this thing🤔

Reply:
Got it @samyakshah3008